resource/systemd/status.go

```diff
@@ -0,0 +1,48 @@
+// Copyright © 2016 Asteris, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package systemd
+
+import "github.com/asteris-llc/converge/resource"
+
+func AppendStatus(a, b *resource.Status) *resource.Status {
```

Comment:
We really shouldn't be doing this kind of thing. Why are you creating more than one Status?